_data/sites.json

```
@@ -17933,6 +17933,20 @@
         ]
     },
 
+    {
+        "name": "Reg.ru",
+        "url": "https://help.reg.ru/support/#request",
+        "difficulty": "medium",
+        "notes": "To deactivate an account in Reg.ru, send a request to the support service with the account's login.",
+        "domains": [
+            "reg.ru",
+            "reg.com",
+            "help.reg.ru",
+            "www.reg.com",
+            "www.reg.ru"
```

```suggestion
        "difficulty": "hard",
        "notes": "To deactivate an account in Reg.ru, send a request to the support service with the account's login.",
        "domains": [
            "reg.ru",
            "reg.com",
            "help.reg.ru"
```
When it needs support involvement we label as hard

---

Apologies! I must have missed that one, I assumed that the issue's author had written it the right way.